include/rocksdb/file_system.h

```diff
@@ -86,14 +76,10 @@ struct IOOptions {
   // Timeout for the operation in microseconds
   std::chrono::microseconds timeout;
 
-  // Priority - high or low
-  IOPriority prio;
```

Comment:
There are a few references to this and rate_limiter_priority in WS code. In order to ease the transition, I suggest marking these fields as "DEPRECATED". We can completely remove these in the next release (7.4), which will give them enough time change their code.